src/scss/responsive/_buttons.scss

```diff
@@ -11,7 +11,7 @@
   }
 }
 
-@media only screen and (max-width: 767px) {
+@media only screen and (max-width: $mobileMax) {
```

Comment:
With sass syntax, we can write better syntax by:

Current syntax:
```
@media only screen and (min-width: $tabletMin) and (max-width: $tabletMax) {
   .btn {
     &--secondary {
       max-height: 60px;
     }
   }
 }

@media only screen and (max-width: $mobileMax) {
  .btn {
     &--secondary {
       max-height: 60px;
     }
  }
}
```

Better syntax group by element:
```
.btn {
  &--secondary {
    @media only screen and (min-width: $tabletMin) and (max-width: $tabletMax) {
       max-height: 60px;
    }
    @media only screen and (max-width: $mobileMax) {
       max-height: 40px;
    }
  }
}
```

Reply:
You can do it on next PR, refactor PR.